projects/mdx/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
i don't think we're should have this, it's inconsistent with the config for other projects and allows confusing imports to be used.